doc/source/release/1.11.0-notes.rst

```diff
@@ -131,108 +131,98 @@ New features
-- `scipy.stats.anderson_ksamp` now performs a permutation version of the test
-  if the user provides argument ``n_resamples``. The is particularly useful
-  when the p-value would be beyond the range of the tabulated values.
-- Improvements to the speed and precision of several statistical distributions.
-  
-  - `scipy.stats.betaprime` ``cdf``, ``ppf``
-  - `scipy.stats.genlogistic` ``logcdf``
-  - `scipy.stats.genhyperbolic` ``cdf`` and ``sf``
-  - `scipy.stats.halfnorm` ``cdf``
-  - `scipy.stats.johnsonsu` ``stats``
-  - `scipy.stats.powerlognorm` ``logsf`` and ``logpdf``
-  - `scipy.stats.dgamma` ``cdf`` and ``ppf``
-  - `scipy.stats.foldnorm` ``cdf``, ``sf``
-
-- Improvements to the ``sf`` and ``isf`` methods of `scipy.stats.dgamma`,
-  `scipy.stats.genlogistic`, `scipy.stats.gibrat`, `scipy.stats.gompertz`,
-  `scipy.stats.halfnorm`, `scipy.stats.johnsonsb`, `scipy.stats.johnsonsu`,
-  `scipy.stats.powerlognorm`, `scipy.stats.powernorm`,
-  `scipy.stats.truncexpon`, `scipy.stats.halflogistic`, `scipy.stats.maxwell`.
-- Improvements to the ``fit`` method of `scipy.stats.lognorm`,
-  `scipy.stats.truncpareto`, and `scipy.stats.vonmises`.
-- Improvements to the ``entropy`` method of several statistical distributions:
-  `scipy.stats.beta`, `scipy.stats.chi`, `scipy.stats.chi2`,
-  `scipy.stats.dweibull`, `scipy.stats.dgamma`, `scipy.stats.f`,
-  `scipy.stats.gamma`, `scipy.stats.gengamma`, `scipy.stats.genlogistic`,
-  `scipy.stats.invamma`, `scipy.stats.invgauss`, `scipy.stats.nakagami`,
-  `scipy.stats.t`, `scipy.stats.truncnorm`, and `scipy.stats.invgamma`.
-- Added support for ``axis``, ``nan_policy``, and masked arrays to
-  `scipy.stats.sem`, `scipy.stats.iqr`.
-- Added the Dirichlet multinomial distribution as
-  `scipy.stats.dirichlet_multinomial`.
-- `scipy.stats.multivariate_t` now has a ``cdf`` method for integrating the
-  PDF within rectangular limits of integration, and a new `entropy` method.
-- Added the ``sf`` method to `scipy.stats.betaprime`.
-- Improved t distribution ``logpdf`` and ``pdf`` for large degrees of freedom.
-- `scipy.stats.rankdata` performance was improved for ``method=ordinal`` and
-  ``method=dense``.
-- `scipy.stats.exponweib` now has improved tail precision.
-- `scipy.stats.dweibull` now has improved precision for positive values.
-- `scipy.stats.monte_carlo_test` now supports multi-sample statistics.
-- Added one-sided confidence intervals to `scipy.stats.bootstrap`.
-- `scipy.stats.geom` ``entropy`` is now calculated analytically.
-- Added `scipy.stats.log_rank` to get a statistical difference between two
-  survival functions.
-- Improved precision of `scipy.stats.anglit`
-- `scipy.stats.moment` now supports non-central moment calculation
-- We now allow a single observation for equal variance in
-  `scipy.stats.ttest_ind`.
-- `scipy.stats.multivariate_normal` and `scipy.stats.invwishart` now have an
-  ``entropy`` method.
-- ``multinomial`` now supports zero trial scenarios
```

Comment:
I removed this intentionally because it is more edge-case maintenance than enhancement.